src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1179,6 +1265,208 @@ static void uct_ib_mlx5_devx_md_cleanup(uct_ib_md_t *ibmd)
     ucs_recursive_spinlock_destroy(&md->dbrec_lock);
 }
 
+#if HAVE_DECL_MLX5DV_DEVX_UMEM_REG_EX
```

Comment:
ifdef

Reply:
actually, it should be if because AC_CHECK_DECLS defines it to 0 if not found

Reply:
fixed